packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -1399,6 +1399,20 @@ export default class Meeting extends StatelessWebexPlugin {
     );
   }
 
+  /**
+   * Post metrics for events, provide the ability to send Call Analyzer for the request of third-party.
+   * @public
+   * @memberof Meeting
+   * @returns {Promise}
+   */
+  // eslint-disable-next-line valid-jsdoc
+  public postMetrics(eventName) {
```

Comment:
type is missing for eventName

Reply:
I meant that the type should be declared in the code and not as a string but the actual possible values of CA events, but I see now that `eventType` is actually just a const object, so this will be problematic. We should change it to be a proper typescript type at some point, but we don't have to do this in this PR.

Maybe in this PR for now at least we could do:
`public postMetrics(eventName: string) {`
and in the jsdoc it should say something like:
`@param {String} eventName - Call Analyzer event, see eventType in src/metrics/config.ts for possible values`

Reply:
that's clean code.